testsuite/features/support/commonlib.rb

```diff
@@ -63,6 +63,11 @@ def compute_image_name
   end
 end
 
+def compute_image_url(image_name)
+  raise "Unknown image #{image_name}" unless image_name == 'suse_os_image'
```

Comment:
Hmm let's ask the others what they think about it.
@Bischoff @lkotek @ktsamis

Reply:
I wonder if using navigation steps (finds and clicks) there is a way to end at the same web page for any image name.

Reply:
To me, passing a parameter that can take only one authorized value does not make much sense. Is it a case of YAGNI?

Reply:
Since the ports were already merged, I will do the same with uyuni.
Feel free to open a card to request a refactor if you wish.

Reply:
Yes, we need a refactor of that. Otherwise the whole refactor is limited to the use of ''suse_os_image' and became useless, fmpov.

Reply:
https://github.com/SUSE/spacewalk/issues/12540